perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -711,18 +711,26 @@ sub bin_services_for_address {
     my $skip_bags = $self->{c}->get_param('skip_bags');
 
     @out = () if $bags_only;
-    my $food_containers = $bags_only ? [ 428 ] : $skip_bags ? [ 424, 423 ] : [ 424, 423, 428 ];
 
-    push @out, {
+    my @food_containers;
+    if ($bags_only) {
+        push @food_containers, 428;
```

Comment:
While we're here, FD 1819 mentions "Already has open 'Food bag request'" so could this check for open 428 too? (and same below).